### ISSUE TITLE
[MASSEMBLY-729] lineEnding ignored when file is not filtered

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-assembly-plugin#940]
**[Tony Jewell](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cregganna)** opened **[MASSEMBLY-729](https://issues.apache.org/jira/browse/MASSEMBLY-729?redirect=false)** and commented

lineEnding=unix is ignored when filtering is not enabled in FileSet.

See attached sample project.

unfiltered.properties - has original (DOS) line endings
filtered.properties - has unix file endings - original had DOS line endings

**NOTE** the pom.xml maven-assembly-plugin:2.5 with the following dependency overrides:

```
<dependency>
	<groupId>org.codehaus.plexus</groupId>
	<artifactId>plexus-archiver</artifactId>
	<version>2.8.2</version>
</dependency>
<dependency>
	<groupId>org.codehaus.plexus</groupId>
	<artifactId>plexus-io</artifactId>
	<version>2.3.3</version>
</dependency>
```

---

**Affects:** 2.5

**Attachments:**
- [maven-assembly-plugin-2.5-lineEnding-ignored-when-not-filtering.zip](https://issues.apache.org/jira/secure/attachment/12714390/maven-assembly-plugin-2.5-lineEnding-ignored-when-not-filtering.zip) (_6.13 kB_)

**Comment by jira-importer** [comment]
**[Kristian Rosenvold](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=krosenvold)** commented

Fixed with a unit test in r1637652, thanks for the report !

**Comment by jira-importer** [comment]
**[Tony Jewell](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cregganna)** commented

Confirmed fixed at head of trunk: 1638029

Thanks Kristian.
